test/models/User.spec.js

```diff
@@ -0,0 +1,94 @@
+import assert from 'power-assert';
+import mockgoose from 'mockgoose';
+import Promise from 'bluebird';
+import UserModel from 'models/User';
+
+describe('/models/User', ()=> {
+  afterEach(()=> {
+    mockgoose.reset();
+  });
+
+  it('同一のメールアドレスは登録できない', (done)=> {
+    const accounts = [
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).